File: 준영/콜라츠 추측.js
function solution(num) {
  var answer = 0
  while (num != 1) {
    if (answer >= 500) {
      break
    }

    if (num % 2) {
      num = (num * 3) + 1
    } else {
      num = parseInt(num / 2)
    }

    answer += 1
  }

  if (num != 1) {
    answer = -1
  }
  return answer
}

File: 준영/프로그래머스/LV1/콜라츠 추측.py
def solution(num):
    answer = 0
    while num != 1:
        if answer >= 500:
            break
        if num % 2:
            num = (num * 3) + 1
        else:
            num //= 2
        answer += 1
    if num != 1:
        answer = -1
    return answer


solution(6)